portfolio/src/main/java/com/google/sps/servlets/data/InitialValues.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.servlets.data;
+
+import java.util.Date;
+
+public final class InitialValues{
```

Comment:
Might be worth a more specific name PageViewData, since while they start as the initial values they get updated directly in this class.

Reply:
Solved